compiler_opt/rl/feature_ops.py

```diff
@@ -80,3 +85,88 @@ def normalize(obs: types.Float):
     return tf.concat(features, axis=-1)
 
   return normalize
+
+
+def get_ir2vec_normalize_fn(ir2vec_with_z_score_normalization: bool = False,
+                            eps: float = 1e-8):
```

Comment:
Return type annotation here?

Reply:
Yeah, added! Other methods in this file do not have type annotations though!